rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -794,6 +794,33 @@ stream {
         {{ $server.ServerSnippet }}
         {{ end }}
 
+        {{ range $errCode := $server.AllCustomHTTPErrors }}
```

Comment:
This still needs to be deduplicated somehow with the template `CUSTOM_ERRORS` above in this file.